sys-apps/portage/files/portage-2.2.18-0001-portage-repository-config.py-add-disabled-attribute-.patch

```diff
@@ -1,51 +0,0 @@
-From 1b8ccc0e487a523aa6d974b3ebe26548ad6502b6 Mon Sep 17 00:00:00 2001
```

Comment:
this is safe to drop with the new portage?

Reply:
It's dropping the `2.2.18` patch; the new version is using the `2.3.8` patch.